Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.12-slim
+LABEL maintainer="m.shmygel@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
Using `COPY . .` can lead to copying unnecessary files into the Docker image. Consider specifying only the necessary files and directories to keep the image size minimal.